### DISCUSSION TITLE
Sonoma-OC-0.9.0-Beta_Release-Infinix-InBook-Series

### DISCUSSION THREAD

**Discussion by devboloji** [devboloji/Infinix-Hackintosh-Opencore-Guide#34] [Discuss about the release]
- Updated all the required kexts. 
- Generate Your SMBIOS if you need iservices like iMessages, etc.
- Bluetooth is ded. Because it's not in latest.
- Fixed trackpad which is not working directly.
- Replaced AirportItlwm with Heliport. Because there is no AirportItlwm kext for Sonoma yet. So install Heliport.app which is on EFI folder. And run Heliport `Run on startup`.
- Everything is same as Previous EFI.
- Removed some kexts which are not making Sonoma bootloop.
**NOTE:  Donot update any kext. Keep it as it should be. Just change SMBIOS and Book picker. DON'T touch anything more than those two. Other wise it may make boot loop.**

- **If you enjoyed this project — or just feeling generous, consider buying me a coffee. Cheers! ☕ :**
# :diamond_shape_with_a_dot_inside:buy me a coffee :coffee:  Donate :  :diamond_shape_with_a_dot_inside:
donate via paypal :
<a href="https://www.paypal.me/369639/"><img src="blue.svg" height="40"></a>  
donate via Paytm:
| <a><img src="darkblue.svg" height="40"></a> |
|-------------------------------------------|
|  `9392656181` |

<hr /><em>This discussion was created from the release <a href='https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide/releases/tag/sonomaEFI_beta'>Sonoma-OC-0.9.0-Beta_Release-Infinix-InBook-Series</a>.</em>

**Comment by subhash-k0** [comment]
Wifi doesn't work, how to resolve it??

**Reply by phuminsingla** [reply]
You can use itlwm.kext + Heliport.app instead for the while.

**Comment by phuminsingla** [comment]
Alternatively, an updated binary is provided by @zxystd https://github.com/OpenIntelWireless/IntelBluetoothFirmware/issues/437#issuecomment-1579931908, which should also be applicable for users updating to macOS Sonoma.

**Reply by devboloji** [reply]
Try it and report me back guys!

**Comment by MonuCarpenter** [comment]
Everything fixed in sonoma? i have infinix x1 i3 10G want to replace my garuda linux to mac os sonoma [single boot] 
Any suggestions newbie here.. i also wanna contribute to this repo in freee time and lets have a coffee after my successfull installation

**Reply by devboloji** [reply]
Tell me your device full name with processor name

**Reply by MonuCarpenter** [reply]
Dedicated Graphic Memory Type | DDR4
-- | --
Processor Brand | Intel
Processor Name | Core i3
Processor Generation | 10th Gen
SSD | Yes
SSD Capacity | 256 GB
RAM | 8 GB
RAM Type | LPDDR4X
Processor Variant | 1005G1
Chipset | Intel Core i3
Clock Speed | 1.2 GHz Max Frequency Upto 3.4GHz
Memory Slots | 1
Graphic Processor | Intel Integrated UHD
Number of Cores | 2
Storage Type | SSD

**Reply by MonuCarpenter** [reply]
last year i had changed the OS to garuda linux

**Reply by MonuCarpenter** [reply]
https://garudalinux.org/

**Reply by devboloji** [reply]
is it fine to talk here or we can go on telegram @Devboloji ?

**Reply by MonuCarpenter** [reply]
Product Link https://www.flipkart.com/infinix-inbook-x1-intel-core-i3-10th-gen-1005g1-8-gb-256-gb-ssd-windows-11-home-xl11-thin-light-laptop/p/itmf6031976051d3?pid=COMG8Z3H5HDHFPHQ&lid=LSTCOMG8Z3H5HDHFPHQSWUHWZ

meanwhile i sent you a message on telegram
